File: src/main/java/es/uji/ei1039/agenda/controller/RootLayout.java
package es.uji.ei1039.agenda.controller;

import javafx.event.ActionEvent;
import javafx.fxml.FXML;

import java.net.URL;
import java.util.ResourceBundle;

public final class RootLayout {

    @FXML // ResourceBundle that was given to the FXMLLoader
    private ResourceBundle resources;

    @FXML // URL location of the FXML file that was given to the FXMLLoader
    private URL location;

    @FXML
        // This method is called by the FXMLLoader when initialization is complete
    void initialize() {
    }

    @FXML
    void handleNew(final ActionEvent event) {
        // TODO
    }

    @FXML
    void handleOpen(final ActionEvent event) {
        // TODO
    }

    @FXML
    void handleSave(final ActionEvent event) {
        // TODO
    }

    @FXML
    void handleQuit(final ActionEvent event) {
        // TODO
    }
}

File: src/main/kotlin/es/uji/ei1039/agenda/view/RootLayout.kt
package es.uji.ei1039.agenda.view

import javafx.application.Platform
import javafx.event.ActionEvent
import javafx.fxml.FXML
import javafx.scene.layout.BorderPane
import tornadofx.View
import tornadofx.get

class RootLayout : View() {

    override val root: BorderPane by fxml(hasControllerAttribute = true)

    init {
        title = messages["title"]

        root.center<ContactOverview>()
    }

    @FXML
    private fun handleNew(event: ActionEvent) {
        // TODO
    }

    @FXML
    private fun handleOpen(event: ActionEvent) {
        // TODO
    }

    @FXML
    private fun handleSave(event: ActionEvent) {
        // TODO
    }

    @FXML
    private fun handleQuit(event: ActionEvent) {
        Platform.exit()
    }
}
